### DISCUSSION TITLE
activity_main.xml

### DISCUSSION THREAD

**Discussion by asraful-me** [jellyfin/jellyfin-android#976] [Q&A] [answered]
<?xml version="1.0" encoding="utf-8"?>
<androidx.fragment.app.FragmentContainerView xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools"
    android:id="@+id/fragment_container"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    tools:layout="@layout/fragment_connect"
    tools:viewBindingIgnore="true" />

7th line error, how can i solved this error?
https://ibb.co/rZwWbmc

**Comment by nielsvanvelzen** [comment, marked as answer]
As we've already told on Matrix. We do not provide support for changing the Jellyfin branding.

**Reply by asraful-me** [reply]
ok I am really sorry

**Reply by cameocoder** [reply]
I'm not aware of the conversation on matrix, but this error is because the fragment_connect layout was removed because the code was converted to Jetpack Compose.

The 'tools:layout="@layout/fragment_connect"' line can be safely removed. 
I did that in this PRs [1660](https://github.com/jellyfin/jellyfin-android/pull/1660) and [1675](https://github.com/jellyfin/jellyfin-android/pull/1675)
